draft-ietf-httpbis-cache-latest.xml

```diff
@@ -531,6 +531,15 @@
    Also, note that unsafe requests might invalidate already-stored responses;
    see <xref target="invalidation" />.
 </t>
+<t>
+   A response that is stored or can be stored can be used to satisfy multiple
+   requests, provided that it is allowed to reuse that response for the requests
```

Comment:
Maybe "A response that is stored or storable can be used to satisfy multiple..." ?